site/components/pages/Blog/Post/index.tsx

```diff
@@ -78,6 +85,56 @@ export const PostPage = (props: PostProps) => {
               {publishedDate}
             </Text>
             <BlockContentRenderer blocks={props.post.body} />
+            {props.post.showDisclaimer && (
+              <div className={styles.disclaimer}>
+                <Text t="caption">
+                  <Trans id="blog.disclaimer.title">Disclaimer:</Trans>
+                </Text>
+                <Text t="caption">
+                  <Trans
+                    id="blog.disclaimer.description"
+                    comment="Long sentence"
+                  >
+                    The information provided in this blog post pertaining to
+                    Klima DAO (“Klima DAO”), its crypto-assets, business assets,
```

Comment:
Just need to update copy to "KlimaDAO" in a few places :)

Reply:
good catch!